src/providers/files-provider.tsx

```diff
@@ -113,6 +112,9 @@ export interface FilesState {
 
   // whether or not the root CID (containing folder for all files) has been published
   rootPublished: boolean
+
+  // Whether or not to provide CIDs to the DHT
+  provideToDHT: boolean
```

Comment:
love this. unfortunately, providing seems to hang forever when this is checked and I add a file.

Also, we will want a `TODO` or issue for "reproviding" to DHT if this is checked after files have already been added.